elfi/methods/bo/utils.py

```diff
@@ -162,3 +162,67 @@ def evaluate_gradient(self, x):
         x = np.atleast_2d(x)
         n, input_dim = x.shape
         return self.scale * self.gradient(x).reshape(n, input_dim)
+
+
+def make_additive_acq(acquisition_class, function):
+    """Make acquisition function adjusted with an additive term.
+
+    Parameters
+    ----------
+    acquisition_class : Type[elfi.methods.bo.acquisition.AcquisitionBase]
+        Acquisition function to be adjusted.
+    function : AdjustmentFunction
+        Function added to the base acquisition function.
+
+    Returns
+    -------
+    AdditiveAcquisition
+
+    """
+    class AdditiveAcquisition(acquisition_class):
+
+        def __init__(self, model, **kwargs):
+            super().__init__(model=model, **kwargs)
+            self._func = function
+
+        def evaluate(self, theta_new, t=None):
+            return super().evaluate(theta_new, t=t) + self._func.evaluate(theta_new)
+
+        def evaluate_gradient(self, theta_new, t=None):
+            t1 = super().evaluate_gradient(theta_new, t=t)
+            t2 = self._func.evaluate_gradient(theta_new)
+            return t1 + t2
+
+    return AdditiveAcquisition
+
+
+def make_multiplicative_acq(acquisition_class, function):
+    """Make acquisition function adjusted with a multiplictive term.
+
+    Parameters
+    ----------
+    acquisition_class : Type[elfi.methods.bo.acquisition.AcquisitionBase]
+        Acquisition function to be adjusted.
+    function : AdjustmentFunction
+        Function that multiplies the base acquisition function.
+
+    Returns
+    -------
+    MultiplicativeAcquisition
+
+    """
+    class MultiplicativeAcquisition(acquisition_class):
```

Comment:
Why put `MultiplicativeAcquisition` inside `make_multiplicative_acq`

Reply:
i'll resolve this on the basis that the issue is discussed on the other thread. ie this is the same as the additive acquisition